infra/helper.py

```diff
@@ -739,6 +741,12 @@ def coverage(args):
 
   run_args = _env_to_docker_args(env)
 
+  if args.port:
```

Comment:
This move is necessary, otherwise i see weird errors
https://stackoverflow.com/questions/48346310/docker-expose-port-oci-runtime-create-failed

Reply:
ack!